### DISCUSSION TITLE
Challenge: Merge Networks Using Different BGP AS Numbers

### DISCUSSION THREAD

**Discussion by ipspace** [bgplab/bgplab#2] [Ideas]
This is as good a place to discuss the [Merge Networks Using Different BGP AS Numbers](https://bgplabs.net/challenge/20-merge-as/) challenge lab as any. Have fun ;)

**Comment by blakedot** [comment]
nice one Ivan!
I think you have a typo in the "IGP" section of the lab page:
"Routers in AS 65004 run IS-IS." should be 
"Routers in AS 65003 run IS-IS."

**Reply by ipspace** [reply]
Thank you, fixed (together with a few other things)
